contracts/auctions/src/lib.rs

```diff
@@ -1,84 +1,21 @@
 #![no_std]
 
-use soroban_sdk::{contract, contractimpl, contracttype, Address, Env, Vec};
-
-#[derive(Clone)]
-#[contracttype]
-pub struct Auction {
-    pub id: u64,
-    pub item_address: Address, // Can be an NFT contract address or a collection contract address
-    pub seller: Address,
-    pub highest_bid: u64,
-    pub highest_bidder: Address,
-    pub buy_now_price: u64,
-    pub end_time: u64,
-    pub status: AuctionStatus,
-}
-
-#[derive(Clone, PartialEq)]
-#[contracttype]
-pub enum AuctionStatus {
-    Active,
-    Ended,
-    Cancelled,
+mod contract;
+mod error;
+mod storage;
+
+#[cfg(test)]
+mod test;
+
+pub mod collection {
+    type NftId = u64;
```

Comment:
Isn't it possible to import it through the hash? I think I did that once.

Reply:
couldn't find any other examples that use custom type from a different contract.

The complier needs to satisfy this:

```
cannot find type `NftId` in this scope
not found in this scope 
```